ios/NewExpensify.xcodeproj/project.pbxproj

```diff
@@ -791,6 +791,7 @@
 				);
 				MTL_ENABLE_DEBUG_INFO = YES;
 				ONLY_ACTIVE_ARCH = YES;
+				REACT_NATIVE_PATH = "${PODS_ROOT}/../../node_modules/react-native";
```

Comment:
This seems fine to me, this has been added here https://github.com/Expensify/App/pull/12570/files and then it must have been removed somewhere again 😵‍💫